account_invoice_accrual/README.rst

```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Account Invoice Accrual
+=======================
+
+Many companies want to establish a complete accounting situation at the end of
+each period. An important aspect of this process is to accrue invoices for
+which the goods or services were received / delivered but for which the invoice
+has not been received / sent.
```

Comment:
The readme is misleading. This module is not related to goods received/delivered not yet invoiced (that's the accrual picking module). This allows to accrue existing draft and pro-forma invoices.

Reply:
I updated the readme (and rebased).